app/components/playfair/CombChart.tsx

```diff
@@ -0,0 +1,150 @@
+import * as d3 from "d3";
```

Comment:
I'm assuming this is ported from the Vue, so I figure we'll do another refactoring pass.